### PR TITLE
[ci] bump elpi to 1.14.1

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
help @coq/ci-maintainers , it seems gtk on 4.13 requires a different package now

**Comment by gares** [comment]
All version of lablgtk3 I have here `conflicts    "base-domains"`,  so I guess we need to disable coqide on edge?

**Comment by gares** [comment]
https://github.com/ocaml/opam-repository/commit/44fc9b9bfaf3dc14cb9f3314c12e96067f959b9b

**Comment by SkySkimmer** [comment]
I don't see how that's relevant. edge is 4.13, not multicore.

**Comment by gares** [comment]
You are right, I got confused

**Comment by gares** [comment]
I tried to undestand how it worked, and I guessed that `opam install foo.1 foo.2` was working, but here it seems not. What did I miss @SkySkimmer ?

File: dev/ci/docker/bionic_coq/Dockerfile [hunk 1]
```diff
@@ -61,16 +61,17 @@ RUN opam switch create "${COMPILER}+32bit" && eval $(opam env) && \
...
+    BASE_OPAM_EDGE="dune.2.9.1 dune-release.1.5.0" \
+    COQIDE_OPAM_EDGE="lablgtk3-sourceview3.3.1.2"
 
 # EDGE+flambda switch, we install CI_OPAM as to be able to use
 # `ci-template-flambda` with everything.
 RUN opam switch create "${COMPILER_EDGE}+flambda" \
       --repositories default,ocaml-beta=git+https://github.com/ocaml/ocaml-beta-repository.git \
       --packages="ocaml-variants.${COMPILER_EDGE}+options,ocaml-option-flambda" && eval $(opam env) && \
-    opam install $BASE_OPAM $BASE_OPAM_EDGE $COQIDE_OPAM $CI_OPAM
+    opam install $BASE_OPAM $BASE_OPAM_EDGE $COQIDE_OPAM $COQIDE_OPAM_EDGE $CI_OPAM
```
Comment (SkySkimmer) [inline dev/ci/docker/bionic_coq/Dockerfile:74]:
```suggestion
    opam install $BASE_OPAM $BASE_OPAM_EDGE $COQIDE_OPAM_EDGE $CI_OPAM
```

Comment (SkySkimmer) [reply dev/ci/docker/bionic_coq/Dockerfile:74]: and put a cairo in COQIDE_OPAM_EDGE

**Comment by SkySkimmer** [comment]
bad cachekey

**Comment by gares** [comment]
it seems it is ok finally

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**